src/prefect/agent/kubernetes/resource_manager.py

```diff
@@ -29,11 +31,10 @@ def __init__(self) -> None:
 
         logger = logging.getLogger("resource-manager")
         logger.setLevel(logging.DEBUG)
-        ch = logging.StreamHandler()
+        ch = logging.StreamHandler(sys.stdout)
         ch.setLevel(logging.DEBUG)
```

Comment:
we can solve this later after this PR, but it seems that this is hard coded to a debugging log level instead of deriving the level from the config